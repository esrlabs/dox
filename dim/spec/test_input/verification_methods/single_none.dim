document: test_verification_methods

ID_1:
  test_setups: none

ID_2:
  verification_methods: none
