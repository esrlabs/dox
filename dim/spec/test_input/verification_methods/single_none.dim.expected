document: test_verification_methods

ID_1:
  verification_methods: none

ID_2:
  verification_methods: none
